### ISSUE TITLE
Training on HMDB51

### ISSUE THREAD

**Issue by Shaoliu089** [alexandrosstergiou/Squeeze-and-Recursion-Temporal-Gates#3]
Hello, thank you for providing the code. I am using your code to train on HMDB-51. However, I found your HMDB-51 models are trained from fine-tuning the HACS pre-trained model.(I got very bad result training on HMDB-51 data only) Can you provide the pre-trained HACS model? Another question: The HACS model is training on the HACS Segments data right? Thank you!

**Comment by alexandrosstergiou** [comment]
Hi @Shaoliu089 ,

We will be releasing the weights in the coming months alongside code updates for stability (mainly for multi-grid training) and bug fixes. I am also looking at possibilities for using single-precision training as apex may be unstable in certain cases (e.g. small gradient values). I will leave this issue open until then.

Best,
Alex

**Comment by alexandrosstergiou** [comment]
Update: You can now initialise your model weights with the provided pre-trained models on HACS from the latest commit [`33bbc7a`](https://github.com/alexandrosstergiou/Squeeze-and-Recursion-Temporal-Gates/tree/33bbc7a3aed42890390d6451902d9776ea727baa)

Best,
Alex
